web-common/src/features/dashboards/url-state/DashboardURLStateSync.svelte

```diff
@@ -174,7 +174,13 @@
   });
 
   async function handleExploreInit(isManualUrlChange: boolean) {
-    if (!exploreSpec || !metricsSpec || initializing) return;
+    if (
+      !exploreSpec ||
+      !metricsSpec ||
+      initializing ||
+      exploreStateFromYAMLConfig.activePage === undefined
+    )
```

Comment:
An explanatory variable could help here too. It should help explain to someone how/why this condition would ever need to be extended in the future.